pyproject.toml

```diff
@@ -18,6 +18,7 @@ requires-python = ">=3.11"
 
 dependencies = [
   "boto3",
+  "orjson"
```

Comment:
This should be an optional dependency in the `npy` group

Reply:
:+1: